packages/neon-invoker/src/index.ts

```diff
@@ -173,30 +176,33 @@ export class NeonInvoker implements Neo3Invoker {
     return await experimental.txHelpers.addFees(trx, config)
   }
 
-  static convertParams(args: Arg[] | undefined): Neon.sc.ContractParam[] {
+  static convertParams(args: ExtendedArg[] | undefined): Neon.sc.ContractParam[] {
     return (args ?? []).map(a => {
+      if (a.type === undefined) throw new Error('Invalid argument type')
+      if (a.value === undefined) throw new Error('Invalid argument value')
+
       switch (a.type) {
         case 'Any':
           return sc.ContractParam.any(a.value)
         case 'String':
-          return sc.ContractParam.string(a.value ?? '')
+          return sc.ContractParam.string(a.value)
         case 'Boolean':
-          return sc.ContractParam.boolean(a.value ?? false)
+          return sc.ContractParam.boolean(a.value)
         case 'PublicKey':
-          return sc.ContractParam.publicKey(a.value ?? '')
+          return sc.ContractParam.publicKey(a.value)
         case 'Address':
+        case 'ScriptHash':
+          return sc.ContractParam.hash160(Neon.u.HexString.fromHex(a.value))
```

Comment:
vc mudou o comportamento do Address mas só pra ser mais abrangente né? Funciona do jeito antigo, correto?

Reply:
It was my error, I forgot that when you don't pass a case block, it runs the next block.